core/src/main/scala/org/apache/spark/broadcast/TorrentBroadcast.scala

```diff
@@ -239,9 +239,9 @@ private[spark] class TorrentBroadcast[T: ClassTag](obj: T, id: Long)
             val estimatedTotalSize = Utils.bytesToString(numBlocks * blockSize)
             logInfo(s"Started reading broadcast variable $id with $numBlocks pieces " +
               s"(estimated total size $estimatedTotalSize)")
-            val startTimeMs = System.currentTimeMillis()
+            val startTimeNs = System.nanoTime()
             val blocks = readBlocks()
-            logInfo("Reading broadcast variable " + id + " took" + Utils.getUsedTimeMs(startTimeMs))
+            logInfo(s"Reading broadcast variable $id took ${Utils.getUsedTimeMs(startTimeNs)}")
```

Comment:
`getUsedTimeMs` sounds like it needs an arg in ms. I know it could mean 'returns a log message that refers to milliseconds', but think it would be clearer to rename the method. I think most people would read it as referring to its argument.

Reply:
ok. I will rename it